### DISCUSSION TITLE
Restrict Allowed Versions based on other Package Version

### DISCUSSION THREAD

**Discussion by SchroederSteffen** [renovatebot/renovate#13837] [Request Help] [answered]
Hi renovate team,

Thanks for the bot! It's an awesome tool!

I've searched for ways to to restrict TypeScript versions based on the used Angular version.
So far, I only found workarounds like restricting the TS version update altogether (e.g. via approvals).

However, I'm wondering if it's possible to write package rules that restrict the allowed versions of a package based on the matched version of another package. I didn't find it in the documentation.

Thanks in advance!

Cheers,
Steffen

**Comment by rarkins** [comment, marked as answer]
This is not currently possible/supported

**Reply by SchroederSteffen** [reply]
That's unfortunate. Thanks for the clarification! 👍
